sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/plan/DefaultPlanCoordinator.java

```diff
@@ -50,15 +50,21 @@ public DefaultPlanCoordinator(
         // with offers first, does not accidentally schedule an asset that's actively being worked upon by another
         // PlanManager that is presented offers later.
         dirtiedAssets.addAll(planManagers.stream()
+                .filter(planManager -> !planManager.getPlan().isWaiting())
                 .flatMap(planManager -> planManager.getDirtyAssets().stream())
                 .collect(Collectors.toList()));
 
         LOGGER.info("Initial dirtied assets: {}", dirtiedAssets);
 
-        for (final PlanManager planManager : planManagers) {
+        for (final PlanManager planManager : getPlanManagers()) {
+            if (planManager.getPlan().isWaiting()) {
+                LOGGER.info("Skipping interrupted plan: {}", planManager.getPlan().getName());
+                continue;
+            }
+
             try {
                 Set<String> relevantDirtyAssets = getRelevantDirtyAssets(planManager, dirtiedAssets);
-                LOGGER.info("Processing offers for plan: {} with relevant dirtied assets: {}.",
+                LOGGER.info("Processing offers for plan: '{}' with relevant dirtied assets: {}.",
                         planManager.getPlan().getName(), relevantDirtyAssets, planManager.getPlan().getName());
```

Comment:
Noticed that this log entry has 3 values but only 2 `{}`s. Looks like the second `getName()` can be removed.